virttest/utils_v2v.py

```diff
@@ -774,12 +774,12 @@ def get_driver_info(self, signed=True):
         while count > 0:
             logging.debug('%d times remaining for getting driver info' % count)
             try:
+                # Clean up output
+                self.session.cmd('cls')
                 output = self.session.cmd_output(cmd)
             except Exception as detail:
                 logging.error(detail)
                 count -= 1
-                # Clean up output
```

Comment:
The 'cls' in 'except' part might be necessary, maybe add another try-except?

Reply:
The 'cls' will run before the 'cmd' each time no matter 'cmd' success or fail. 
This can cover the failed scenario in original code.
@chloerh

Reply:
ok